README.md

```diff
@@ -12,49 +16,146 @@ npm i @autotelic/fastify-hmac
 
 ## Usage
 
-- Require the plugin
-- Provide configuration object that contains the following:
-  - A `sharedSecret` string
-  - A `getAlgorithm` function that returns an algorithm string
-  - A `getDigest` function that returns a digest string
-
-### Example:
+- Register plugin. This will decorate your `fastify` instance with a request method `HMACValidate`.
+  - During registration, provide a configuration object that contains the following:
+    - A `sharedSecret` string
+    - A `getAlgorithm` function that returns an algorithm string
+    - A `getDigest` function that returns a digest string
```

Comment:
Lets update these options now that getDigest has been implemented. 👍

Reply:
I updated the naming and added details for `getDigest` and `digestEncoding`.